examples/connectivity/plot_extract_regions_statistical_maps.py

```diff
@@ -0,0 +1,73 @@
+"""
+Region Extraction using a t-statistical map (3D)
+================================================
+
+This example shows how to extract regions or separate the regions
+from a statistical map.
+
+We use localizer t-statistic maps from :func:`nilearn.datasets.fetch_localizer_contrasts`
+as an input image.
+
+The idea is to threshold an image to get foreground objects using a
+function :func:`nilearn.image.threshold_img` and extract objects using a function
+:func:`nilearn.regions.connected_regions`.
+"""
+
+################################################################################
+# Fetching t-statistic image of localizer constrasts by loading from datasets
+# utilities
+from nilearn import datasets
+
+n_subjects = 3
+localizer_path = datasets.fetch_localizer_contrasts(
+    ['calculation (auditory cue)'], n_subjects=n_subjects, get_tmaps=True)
```

Comment:
Can you define a variable here 'tmap_filename = localizer_path.tmpas[2]' and use it below.